src/pages/index.js

```diff
@@ -85,32 +146,127 @@ function handleAddCardSubmit(newCardData, cardListElement) {
   newCardPopup.close();
 }
 
+function handleCardDeleteClick(id) {
+  deleteConfirm.open(id);
+}
+
 function createCard(cardData) {
-  return new Card(cardData, "#card-template", handleImageClick).getView();
+  return new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleCardLike,
+    handleDelete
+  ).getView();
+}
+
+const cardSection = new Section(
+  {
+    renderer: (item) => {
+      cardSection.addItem(makeCard(item));
+    },
+  },
+  ".cards__list"
+);
+
+// Handle card delete //
+function handleDelete(card) {
+  deleteConfirm.open();
+  deleteConfirm.setSubmitAction(() => {
+    deleteConfirm.renderLoading(true);
+    api
+      .deleteCard(card._id)
+      .then(() => {
+        card.removeCard();
+        deleteConfirm.close();
+      })
+      .catch(console.error);
+  });
+}
+
+// Handle Like click //
+function handleCardLike(card) {
+  if (card._isLiked) {
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.